AssignmentOne.java

```diff
@@ -0,0 +1,15 @@
+
+public class AssignmentOne {
+
+	public static void main(String[] args) {
+		Q1 q1 = new Q1();
+		
+		System.out.println("Expected false, returned " + q1.isPermutation(null, null));
+		System.out.println("Expected false, returned " + q1.isPermutation("", null));
+		System.out.println("Expected false, returned " + q1.isPermutation(null, "HelLo"));
+		System.out.println("Expected false, returned " + q1.isPermutation("Hello", "Helo"));
+		System.out.println("Expected true, returned " + q1.isPermutation("Hello", "HelLo"));
```

Comment:
Also a test case where the letters are shuffled would be good, "Hello" vs "elloh"